test/std-test.cc

```diff
@@ -90,6 +90,34 @@ TEST(std_test, optional) {
 #endif
 }
 
+namespace my_nso {
+enum class my_number {
+  one,
+  two,
+};
+auto format_as(my_number number) {
```

Comment:
Here and below a return type is needed.

Reply:
> Here and below a return type is needed.
Added now. I am spoiled by working with C++20...